### ISSUE TITLE
Add BSWH to All Connect Metric Reports + Weekly Log

### ISSUE THREAD

**Issue by robertsamm** [Analyticsphere/metricsReportsRequests#145]
In preparation for BSWH launch in June, we will need to add BSWH to all metric reports that are run including the weekly Operations Report, Biospecimen Report, Survey reports, monthly Duplicates Report, Reported Cancer Hx in UP Report, etc. In addition, they should be add to the overall Weekly Logs that capture all sites data and have a weekly log created specifically for their site and added to their site folder.

For the metrics that stratify by site, please list BSWH first/at the top. Note, BSWH is only launching with research collections and are not doing clinical collections at this time.

Planned timeline for June release to plan for when these should all be added by and to test reports using stage data below.
Stage Testing: 6/17 - 6/21
Prod Release: 6/24 (10am ET)

Please let me know if you have any questions!

**Comment by KELSEYDOWLING7** [comment]
Thanks for giving me the testing dates! The code should be about done (Weekly Biospecimen, Weekly Module Metrics, Weekly Operations Report, Weekly PROMIS Completion vs Notifications Report, Monthly Data Destruction Report,  Monthly Cancer History, and Monthly Duplicates Report). I'll test it once we have stage data.

Do we expect the site to be added to this variable at some point? BioShip_LogSite_v1r0

**Comment by brotzmanmj** [comment]
It should be, as this is needed for the functioning of the shipping dashboard. @mnataraj92 please check with Gene as to whether this has already been done but just hasn't been documented in the DD. It would be the same CID for BSWH the we use for variable 827220437

**Comment by mnataraj92** [comment]
Confirmed with Gene that it's been done for BioShip_LogSite_v1r0, it just wasn't requested to be added to the DD. @rebexxxxxx can you please add this to the DD change log to add BSWH as an option for BioShip_LogSite_v1r0?

**Comment by rebexxxxxx** [comment]
BSWH has been added as a category to the BioShip_LogSite_v1r0 variable on the DD change log!

**Comment by KELSEYDOWLING7** [comment]
Testing these all in prod now that we have data.

Can you please send me the Box folder number/link to create the log?

**Comment by brotzmanmj** [comment]
Hi, I created this folder for the BSWH Reports https://nih.app.box.com/folder/271947385544

The subfolder for the BSWH Weekly Log is https://nih.app.box.com/folder/271950400797

The subfolder for BSWH refusal and withdrawal report is https://nih.app.box.com/folder/271949968140

**Comment by KELSEYDOWLING7** [comment]
- [x] Weekly Operations Report
- [x] Weekly Logs
- [x] Monthly Duplicates Report
- [x] Monthly Cancer History Report
- [x] Weekly Refusal and Withdrawal Report
- [x] Biospecimen Report
- [x] Biospecimen QC
- [x] Weekly Module Metrics
- [x] Weekly PROMIS Survey
- [x] Monthly Data Destruction
- [x] Module QC
- [x] Recruitment QC

**Comment by KELSEYDOWLING7** [comment]
@robertsamm This should be completed with all weekly and monthly reports having run on Monday. Please let me know if anything else is needed or if this is good to close

**Comment by KELSEYDOWLING7** [comment]
@robertsamm Is this issue good to close? The few column issues on the Operations report that I emailed about last week have been resolved, though most of those tables we'll be deleted anyways with the Operations report revamp

**Comment by robertsamm** [comment]
Hi @KELSEYDOWLING7 can we keep this issue open until the Operations Report is revamped? When we review the finalized updated report we'll also review for BSWH. Hopefully by then we will also have more BSWH data to fill in some of those blanks as well.

**Comment by erincschwartz** [comment]
@KELSEYDOWLING7 I am reviewing the metrics report from Monday. Can you add in the breakdown by location for the BSWH research collections to Table 1.2? I believe we discussed, are you waiting on anything from me?

**Comment by KELSEYDOWLING7** [comment]
@robertsamm That's fine!

@erincschwartz  Yes. That's now been added and it'll be applied to Monday's report

**Comment by KELSEYDOWLING7** [comment]
This should be completed as of the latest Ops report run. Report posted in #156 . The weekly logs have been maintained despite the Ops report on hold
